tasks/strings/strings.js

```diff
@@ -4,23 +4,40 @@
  * @param {string} str
  */
 function reverseString(str) {
+  if (typeof str !== "string") {
+    return 'This is not a string!';
+  }
+  const stringArr = str.split("");
+  stringArr.reverse();
+  return stringArr.join("");
 }
 
 /**
  * 1. Calculate a century by given year
  * if a number is passed
-* @param {number} year
-*/
+ * @param {number} year
+ */
 function centuryFromYear(year) {
-
+  const yearAsInt = parseInt(year);
+  centuryPart = Math.floor(yearAsInt / 100);
```

Comment:
Nope, please use Math.ceil() and you won't need any additional actions

Reply:
Thanks, it would be much better.